tests/API/APIRestTest.php

```diff
@@ -423,18 +441,20 @@ public function testBadEndpoint($session_token) {
          $res = $this->doHttpRequest('GET', 'badEndpoint/',
                                             ['headers' => [
                                              'Session-Token' => $session_token]]);
+         $this->assertFalse(true); // if HTTP 2xx then the test must fail
```

Comment:
I understand your goal, but if the response is 200 the test will succeed and this is not what we want.

Maybe  the following would be more appropriate ?
$this->assertGreaterThanOrEqual(400, $res->getStatusCode());